backend/apps/events/validators.py

```diff
@@ -0,0 +1,244 @@
+    """
+    Checks that binding signup is True if a price has been set on the event
+    """
+    if attendable is not None:
+        # Attendable already exists, may exist values for price and/or binding_signup already as well
+        price = None
+        if hasattr(attendable_data, "price") and attendable_data.price is not None:
+            price = attendable_data.price
+
+        elif hasattr(attendable, "price"):
+            price = attendable.price
+
+        binding_signup = attendable.binding_signup
+        if hasattr(attendable_data, "binding_signup"):
+            binding_signup = attendable_data.binding_signup
+
+        if price is not None and not binding_signup:
+            raise ValidationError("Betalt påmelding krever bindende påmelding")
+    else:
+        if (
+            hasattr(attendable_data, "price")
+            and attendable_data.price is not None
+            and hasattr(attendable_data, "binding_signup")
+            and not attendable_data.binding_signup
+        ):
+            raise ValidationError("Betalt påmelding krever bindende påmelding")
+
+
+def slot_distribution_validation(total_allowed_grade_years, slot_distribution, total_available_slots):
+    """
+    Checks whether the grade groups in the slot distribution dict are the same as the
+    allowed grade years on the event.
+    """
+
+    # Make allowed grades a sorted string without commas, e.g. "124"
+    allowed_grades = ""
+    total_slots = 0
+    for slot_dist_element in slot_distribution:
+        allowed_grades += slot_dist_element.grade_group + ","
+        total_slots += slot_dist_element.available_slots
+
+    allowed_grades = allowed_grades[:-1]
+    sorted_grades = sorted(allowed_grades)
+    string_sorted_grades = "".join(sorted_grades).replace(",", "")
+
+    # Do the same for total_allowed_grade_years and check that the two string are equal
+    if string_sorted_grades != "".join(str(grade) for grade in total_allowed_grade_years):
+        raise ValidationError(
+            "Trinnene arrangementet er åpent for stemmer ikke overens med plass fordelingen for trinn"
+        )
+
+    # Make sure that the sum of available slots in the slot distribution is not fewer than total available slots field
+    if total_slots < total_available_slots:
+        raise ValidationError("Kan ikke ha færre plasser i plassfordelingen enn det er plasser på arrangementet")
+
+    # Make sure each grade year is in at most one grade group
+    listed_total_grades_allowed: list[int] = [int(val) for val in allowed_grades.split(",")]
+    if len(set(listed_total_grades_allowed)) != len(listed_total_grades_allowed):
+        raise ValidationError("Samme trinn kan ikke være i flere 'Antall plasser'-kategorier")
+
+
+def create_event_validation(event_data, attendable_data=None):
+    """
+    Validates input data for create event mutation
+    """
+    end_time = None
+    if event_data.end_time:
+        end_time = event_data.end_time
```

Comment:
Could we just go like this?
```suggestion
    end_time = event_data.end_time
```